### ISSUE TITLE
Issue with Symfony - Unable to Launch New Process

### ISSUE THREAD

**Issue by frisch1** [norkunas/youtube-dl-php#49]
Hello...

This issue cropped up a week ago. Not sure what causes it. When youtube-dl fires up through Norkunas/youtube-dl-php, I get:
` ["message":protected]=>
  string(31) "Unable to launch a new process."
  ["string":"Exception":private]=>
  string(0) ""
  ["code":protected]=>
  int(0)
  ["file":protected]=>
  string(60) "/home/ubuntu/php_composer/vendor/symfony/process/Process.php"
  ["line":protected]=>
  int(288)
  ["trace":"Exception":private]=>
  array(4) {
    [0]=>
    array(6) {
      ["file"]=>
      string(60) "/home/ubuntu/php_composer/vendor/symfony/process/Process.php"
      ["line"]=>
      int(196)
      ["function"]=>
      string(5) "start"
      ["class"]=>
      string(33) "Symfony\Component\Process\Process"
      ["type"]=>
      string(2) "->"
      ["args"]=>
      array(1) {
        [0]=>
        NULL
      }
    }
    [1]=>
    array(6) {
      ["file"]=>
      string(60) "/home/ubuntu/php_composer/vendor/symfony/process/Process.php"
      ["line"]=>
      int(220)
      ["function"]=>
      string(3) "run"
      ["class"]=>
      string(33) "Symfony\Component\Process\Process"
      ["type"]=>
      string(2) "->"
      ["args"]=>
      array(1) {
        [0]=>
        NULL
      }
    }
    [2]=>
    array(6) {
      ["file"]=>
      string(74) "/home/ubuntu/php_composer/vendor/norkunas/youtube-dl-php/src/YoutubeDl.php"
      ["line"]=>
      int(241)
      ["function"]=>
      string(7) "mustRun"
      ["class"]=>
      string(33) "Symfony\Component\Process\Process"
      ["type"]=>
      string(2) "->"
      ["args"]=>
      array(1) {
        [0]=>
        NULL
      }
    }
    [3]=>
    array(6) {
      ["file"]=>
      string(42) "/home/ubuntu/api/api-word-timing.php"
      ["line"]=>
      int(61)
      ["function"]=>
      string(8) "download"
      ["class"]=>
      string(19) "YoutubeDl\YoutubeDl"
      ["type"]=>
      string(2) "->"
      ["args"]=>
      array(1) {
        [0]=>
        string(43) "https://www.youtube.com/watch?v=sfOM27C0MLU"
      }
    }
  }
  ["previous":"Exception":private]=>
  NULL
}
`

At this point, I've tried:
-- Removing Symfony (but can't remove the dependencies in youtube-dl-php
-- Changing symfony versions from 2.8 to 3.3 and back (same error)

I'm running Ubuntu 16.04, youtube-dl version is 2017-11-06 (but also errors on earlier versions the same way, I checked). Installed youtube-dl-php via composer.

Any suggestions? I lean on this a lot, but not sure when Symfony went bonkers on me...

**Comment by norkunas** [comment]
Hey,

Are you on v0.1 or dev-master?

**Comment by frisch1** [comment]
Hi... Thanks for the reply. Unsure... I'm installing via composer:
composer require norkunas/youtube-dl-php

... so not specifying via a git clone. But I'm installing as ^0.3.5...

FWIW, I was able to sort of hack around. Still some odd messages like:
 string(186) "exec 'youtube-dl' 'https://www.youtube.com/watch?v=VovOVqyK3to' '--no-playlist' '--print-json' '--ignore-config' '--continue' '--format=bestaudio[ext=m4a]/mp4' '--output=VovOVqyK3to.m4a'"

... from symfony that haven't cropped up before. What I did:
-- Removed youtube-dl-php from composer.json and updated
-- Removed symfony manually
-- Removed "sami/sami": "~1.0",
-- Updated symfony manually to the latest version via composer.json 
-- re-added youtube-dl-php

...and  other than some odd messages, it seems to be working again. However, before the beginning of the month, haven't had an issue since we've had our processes running since the spring. Not sure what's clashing, but something this...
